### PR TITLE
Code

### PR DESCRIPTION
**Pull request by ABohynDOE**
I've finished the `csv2json.py`script that converts a csv file to a JSON script and I've added some more documentation

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Glossary
+
+Glossary of all experiment attributes with short definitions.
+
+* **Fractional factorial**
+  : A design that is the fraction of a full-factorial design.
+  A full-factorial $2^{k}$ contains all combinations of the levels of $k$ factors.
```
Comment (dmbarreiro) [inline docs/glossary.md:7]: github's markdown does not understand late syntax, that's a pity. There are some solutions out there but nothing we can directly use. We can talk about it tomorrow but for the moment I find this OK

**Review by dmbarreiro — CHANGES_REQUESTED** [review]

File: utils/csv2json.py [hunk 2]
```diff
@@ -1,87 +1,195 @@
 """
```
[the remark this answers is not in the document]
Comment (ABohynDOE) [reply utils/csv2json.py:1]: Good idea

**Review by dmbarreiro — APPROVED** [review]
All good for me, let's wait for @saif-analyze or @mariobecerra  review of csv2json.py
Good job!

**Comment by dmbarreiro** [comment]
I will merge this already, if something needs to be adjusted we will do it later

**Pull request merged**